angular-cli.json

```diff
@@ -50,6 +59,17 @@
   "defaults": {
     "styleExt": "scss",
     "prefixInterfaces": false,
-    "lazyRoutePrefix": "+"
```

Comment:
Yeah, we need to remove those `+` from the names haha
We just had it that way because this was created a long time ago when `+` had a "meaning"